src/thor/map_matcher.cc

```diff
@@ -164,9 +151,27 @@ MapMatcher::FormPath(meili::MapMatcher* matcher,
     }
   }
 
-  // Interpolate match results if using timestamps for elapsed time
-  std::list<std::vector<interpolation_t>> interpolations;
+  return origin_epoch;
+}
+
+// We can return multiple paths from here. Any time a MatchResult (trace point) is marked as a
+// break point or break_through point we will make a new leg and hence a new vector of path infos
+// also whenever there is a discontinuity there will be a new vector of path infos created
+std::deque<std::pair<std::vector<PathInfo>, std::vector<const meili::EdgeSegment*>>>
+MapMatcher::FormPath(meili::MapMatcher* matcher,
```

Comment:
all of the insanity that used to happen in trace_route_action.cc is now easily handled in this function